ui/src/app/features/app-user-list/application-user-list.component.ts

```diff
@@ -136,6 +136,11 @@ export class ApplicationUserListComponent implements OnInit, OnDestroy {
       return;
     }
 
-    this.snackBarService.openInfo(void 0, void 0, `Invitation was sent to ${email}`);
+    this.snackBarService.openNotification(
+      {
+        messageText: 'application_user_list.invitation_sent',
+        messageOptions: {email: `${email}`}
```

Comment:
redundant string interpolation {email: `${email}`}
change to messageOptions: { email }
Please also enable prettier formatter in your IDE